sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -243,6 +251,7 @@
       abstract Builder setQuery(Query query);
       abstract Builder setNamespace(String namespace);
       abstract Builder setNumQuerySplits(int numQuerySplits);
+      abstract Builder setLocalhost(String hostPort);
```

Comment:
I would have used 'InetAddress' but to keep it in sync with the DatastoreOptions which uses a String, I kept it consistent.